chunking_test.py

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+
+import zarr
+
+store = zarr.DirectoryStore("data/chunking_test.zarr")
+z = zarr.zeros((20, 3), chunks=(3, 3), shards=(2, 2), store=store, overwrite=True, compressor=None)
```

Comment:
`shards` is specified in units of chunks?

Reply:
Yes :+1: